LadderBots.json

```diff
@@ -0,0 +1,24 @@
+{
+	"Bots": {
+		"ProtossBot": {
+			"Race": "Protoss",		\\Race of your bot
+			"Type": "BinaryCpp",	\\Type (BinaryCpp or Python)
+			"RootPath": "C:/Ladder/Bots/",	\\Path to the bot file
+			"FileName": "ProtossBot.exe",		\\Name of the bot file
+			"Args": "--arg1 --arg2"			\\Optional command line arguments that gets passed to the bot file.
```

Comment:
Is it just me or are these comment slashes the wrong way around?
@Cryptyc can probably just fix these in github when he merges.

Reply:
Fixed.